process/templates/PROJECT_NAME_sandbox_stage.md

```diff
@@ -0,0 +1,25 @@
+## Application for creating a new project at Sandbox stage
+
+### List of project maintainers
+The project must have a minimum of two maintainers with different company affiliations.
```

Comment:
Suggest "organizational" instead of "company" for reasons discussed before.

Reply:
Addressed in PR#117

Reply:
Since PR#117 is landed, should we update and resolve this?